src/screens/BeerOfTheDay/BeerOfTheDay.jsx

```diff
@@ -1,53 +1,49 @@
-import React, { useState } from 'react';
-import { fetchRandomBeer } from '../../services/beers';
-import Heading from '../../shared/components/Heading/Heading';
-import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../services/localStorage';
-import Beer from '../../shared/components/Beer/Beer';
-import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteButton';
-import Button from '../../shared/components/Button/Button';
-import './styles.scss';
-
-
-
+import React, { useState } from 'react'
+import { fetchRandomBeer } from '../../services/beers'
+import Heading from '../../shared/components/Heading/Heading'
+import {
+  saveItemInLocalStorage,
+  getItemFromLocalStorage,
+} from '../../services/localStorage'
+import Beer from '../../shared/components/Beer/Beer'
+import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteButton'
+import Button from '../../shared/components/Button/Button'
+import './styles.scss'
 
 function BeerOfTheDayComponent() {
-
   const [beer, setBeer] = useState({
     name: '',
     abv: '',
     img: '',
     ibu: '',
-    isPlaceholder: true
+    description: '',
+    isPlaceholder: true,
```

Comment:
Do you need this?

Reply:
Yes because otherwise I see beer before clicking the button